packages/a11y/src/index.ts

```diff
@@ -21,52 +23,58 @@ function getWebpackConfig(p: string) {
     .getConfig();
 }
 
-function printResults(results: axe.AxeResults, impact: number): string {
+function formatResults(results: IResult[], impact: number): string {
   const msg: string[] = [];
   let index = 1;
-  results.violations.forEach((violation) => {
-    if (impactArray.indexOf(violation.impact) + 1 >= impact) {
-      violation.nodes.forEach((node) => {
-        msg.push(`${index++}. ${chalk.red(violation.id === 'duplicate-id' ? 'Document' : node.target[0])}: (Impact: ${violation.impact}) ${node.failureSummary}`);
+  results.forEach((res) => {
+    if (res.error) {
+      msg.push(`${index++}. ${res.comp}: Error while testing component - ${res.error}`);
+    } else if (res.result) {
+      res.result.violations.forEach((violation) => {
+        if (impactArray.indexOf(violation.impact) + 1 >= impact) {
+          violation.nodes.forEach((node) => {
+            const compName = (`${res.comp} - ${node.target[0]}`);
```

Comment:
It's not clear what `node.target[0]` is, and giving it a name would help. Something like `const cowsPerCubicMeter = node.target[0]`.

Is it possible that there's going to be 0 or more than one targets?